easybuild/easyconfigs/q/Qt/Qt-4.8.7-CrayGNU-17.08.eb

```diff
@@ -0,0 +1,26 @@
+name = 'Qt'
+version = '4.8.7'
+# https://github.com/hpcugent/easybuild-easyconfigs/issues/2777
+#gcc434:ko version = '4.8.7'
+
+homepage = 'http://qt-project.org/'
+description = "Qt is a comprehensive cross-platform C++ application framework."
+
+toolchain = {'name': 'CrayGNU', 'version': '17.08'}
+
+source_urls = [ 'http://download.qt.io/official_releases/qt/4.8/4.8.7/' ]
```

Comment:
Please consider using variables for `shortversion` and `version` in the download link, to reduce the possibility of mistakes when updating the recipe in the future:
`http://download.qt.io/official_releases/qt/%(shortversion)s/%(version)s/`

Reply:
will do. Is this explained somewhere ? could not find it online